visp_tracker/include/visp_tracker/tracker-viewer.h

```diff
@@ -80,8 +80,8 @@ namespace visp_tracker
     (const sensor_msgs::msg::Image::ConstPtr& imageConst,
      const sensor_msgs::msg::CameraInfo::ConstSharedPtr& infoConst,
      const geometry_msgs::msg::PoseWithCovarianceStamped::ConstPtr& trackingResult,
-     const visp_tracker::MovingEdgeSites::ConstPtr& sitesConst,
-     const visp_tracker::KltPoints::ConstPtr& kltConst);
+     const visp_tracker::msg::MovingEdgeSites::ConstPtr& sitesConst,
+     const visp_tracker::msg::KltPoints::ConstPtr& kltConst);
 
```

Comment:
Vu ici https://design.ros2.org/articles/generated_interfaces_cpp.html
For similarity to ROS 1 the typedefs Ptr and ConstPtr still exist but are deprecated.

Il faudrait donc remplacer `ConstPtr` par `ConstSharedPtr`partout